### PR TITLE
add more bash completions

### PR DESCRIPTION
**Pull request by baude**
Resolves: #660

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
@rhatdan I couldnt get the inspect to list both images and containers ... mind peeking at it?

**Comment by rhatdan** [comment]
LGTM
Lets get this merged and then I can look at the inspect issue.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @umohnani8 @giuseppe PTAL

File: completions/bash/podman [hunk 1]
```diff
@@ -1427,24 +1500,30 @@ _podman_save() {
 	 -q
 	 --quiet
      "
-     _complete_ "$options_with_args" "$boolean_options"
-}
 
-_podman_export() {
```
Comment (mheon) [inline completions/bash/podman]: Did we just have duplicates for `podman export` bash completion?

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**